### PR TITLE
[native] Add e2e test for binomial_cdf and cauchy_cdf functions

### PR DESCRIPTION
**Pull request by gopukrishnasIBM**
Test plan - Added e2e test for new scalar functions binomial_cdf and cauchy_cdf.
PR for binomial_cdf : https://github.com/facebookincubator/velox/pull/4686
PR for cauchy_cdf : https://github.com/facebookincubator/velox/pull/4935
```
== RELEASE NOTES ==
```

### CODE REVIEW

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java [hunk 1]
```diff
@@ -1223,6 +1223,16 @@ public void testLambda()
         assertQuery("select transform(x, i->transform(i, j->j*y)) from (select x, y*y as y from (values row(array[array[1]], 2)) t(x, y))");
     }
 
+    @Test
+    public void testCDF()
+    {
+        assertQuery("SELECT binomial_cdf(CAST (nationkey AS INTEGER), 0.1, 0 ) FROM nation WHERE nationkey > 0");
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:1229]: Please can you check if these CASTs are necessary on the Presto-java side also ( i.e. Does the query without the casting work ?)

Comment (gopukrishnasIBM) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:1229]: @aditi-pandit CASTs are necessary, checked on the Java side.

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java [hunk 2]
```diff
@@ -1223,6 +1223,16 @@ public void testLambda()
         assertQuery("select transform(x, i->transform(i, j->j*y)) from (select x, y*y as y from (values row(array[array[1]], 2)) t(x, y))");
     }
 
+    @Test
```
[the remark this answers is not in the document]
Comment (gopukrishnasIBM) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java]: @aditi-pandit Moved to AbstractTestNativeProbabilityFunctionQueries file.

**Review by aditi-pandit — APPROVED** [review]

**Comment by aditi-pandit** [comment]
@gopukrishnasIBM : Please rebase your code and push again. We can merge after a clean test run.

**Comment by gopukrishnasIBM** [comment]
@aditi-pandit Rebase done. All checks passed.

**Pull request merged**